doc/internals/extending-xarray.rst

```diff
@@ -88,9 +88,21 @@ Back in an interactive IPython session, we can use these properties:
 
 The intent here is that libraries that extend xarray could add such an accessor
 to implement subclass specific functionality rather than using actual subclasses
-or patching in a large number of domain specific methods. For further reading
-on ways to write new accessors and the philosophy behind the approach, see
-:issue:`1080`.
+or patching in a large number of domain specific methods.
+
+.. note::
+
+   With the exception of ``__init__``, ``__del__`` and the descriptor protocols it is
```

Comment:
is that right? I assumed that the descriptor protocols would not work because the accessor object is constructed by another descriptor, but `__init__` is only restricted in the signature (this could probably use more careful wording) and I don't know if it makes sense to discourage using `__del__`?